src/components/pricing/sections/comparison.ts

```diff
@@ -21,14 +21,14 @@ export const FEATURES: Record<string, Feature> = {
   },
   recordings: {
     name: 'Uploaded recordings',
-    description: `By default all failed tests are uploaded, but we recommend also uploading all recordings for atleast one run a day so you have a reference point.`,
-    learnMore: '#'
+    description: `By default all tests are uploaded, but it's easy to upload a subset of recordings to stay within the upload limits.`,
+    learnMore: 'https://docs.replay.io/ci-workflows/upload-strategies'
   },
   processed: {
-    name: 'Replayed recordings',
+    name: 'Instant replays',
```

Comment:
I like this change. I think it's a much better term than "replayed recordings"

I think we might still want a little info (i) icon with a tooltip explaining what an "instant replay" is

Reply:
We have a tooltip, but agreed that an info icon could help

Reply:
I think that's a pretty important follow up. I had no idea there was a tooltip, so I bet a lot of other people would overlook this too.